.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,9 @@
-**This project uses [semver](http://semver.org), please check the scope of this pr:**
+### This project uses [semver](semver.org), please check the scope of this pr:
```

Comment:
@juwara0 @notmessenger we should agree on ** or ###. `ember-frost-core` is using ** and `ember-frost-list` is using `###`.